WolvenKit.App/Services/PluginService.cs

```diff
@@ -134,6 +167,33 @@ public async Task CheckForUpdatesAsync()
         {
             foreach (var plugin in Plugins)
             {
+                if (plugin.Id == EPlugin.redmod)
```

Comment:
How does this check for updates? I am not following it. Seems like the remote version is always `1.0`.

Reply:
this is a todo